### ISSUE TITLE
Does TFRS support personalized ranking?

### ISSUE THREAD

**Issue by dgoldenberg-audiomack** [tensorflow/recommenders#303]
Question: does TFRS support personalized ranking? Given a list of items and a user ID, this would return the item list re-ordered in the fashion most relevant/appropriate for the given user.

I don't seem to see this in the framework. If it's not available, I'd like this to be an enhancement request to get it added to TFRS.

**Comment by maciejkula** [comment]
Of course! Have you seen https://www.tensorflow.org/recommenders/examples/basic_ranking?

**Comment by maciejkula** [comment]
To add a bit more detail: in your model, you'd broadcast the user id to the shape of the candidates, then concatenate them and perform the prediction computation from there.
